packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/connector/ConnectorContextualMenu.tsx

```diff
@@ -174,13 +174,13 @@ export const ConnectorContextualMenu = ({}: ConnectorContextualMenuProps) => {
       }
     }
     if (invokeSingleClickOnTwoDiagramElementToolError) {
-      addErrorMessage(error.message);
+      addErrorMessage(error?.message ?? 'An unexpected error has occurred, please refresh the page');
```

Comment:
This should be `invokeSingleClickOnTwoDiagramElementToolError.message` and it cannot be null / undefined since we just tested it.

Reply:
Done